src/Tribe/Template.php

```diff
@@ -767,6 +760,36 @@ public function template( $name, $context = [], $echo = true ) {
 		return $html;
 	}
 
+	/**
+	 * Includes a give PHP inside of a safe context.
+	 *
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: f8bf36e7431dbefba88f6d92dfb458be</sub>